client/src/pages/private/SiteParticipantsTable.js

```diff
@@ -98,138 +170,25 @@ export default ({ id, siteId, onArchiveParticipantAction, stale, setStale }) =>
     });
   }, [dispatch, id]);
 
-  useEffect(() => {
-    if (stale) {
-      dispatch({
-        type: SiteDetailTabContext.types.LOAD_SITE,
-        payload: {},
-      });
-      fetchDetails(id).then((response) => {
-        dispatch({
-          type: SiteDetailTabContext.types.UPDATE_SITE,
-          payload: { site: response },
-        });
-      });
-      setStale(false);
-    }
-  }, [dispatch, id, stale, setStale]);
-
   useEffect(() => {
     dispatch({
       type: SiteDetailTabContext.types.SELECT_TAB,
       payload: { tab: tabs[0], roles },
     });
   }, [dispatch, roles, siteId]);
 
-  const [orderBy, setOrderBy] = useState(columns[4]?.id || 'participantName');
-  const handleRequestSort = (event, property) => {
-    const isAsc = orderBy === property && order === 'asc';
-    setOrder(isAsc ? 'desc' : 'asc');
-    setOrderBy(property);
-  };
-
-  const sort = (array) => _orderBy(array, [orderBy, 'operatorName'], [order]);
+  useEffect(() => {
+    setRows(selectedTab === 'Hired Participants' ? fetchedHiredRows : fetchedWithdrawnRows);
```

Comment:
what about the `Site Details` page?

Reply:
Replaced with switch statement in last commit. Refactored a bit to use constants so I wasn't adding _more_ usage of string literals.